tools/exp/go.mod

```diff
@@ -4,6 +4,7 @@ go 1.19
 
 require (
 	github.com/aiven/aiven-go-client v0.0.0
+	github.com/google/go-cmp v0.5.8
```

Comment:
from https://pkg.go.dev/github.com/google/go-cmp/cmp:

> This package is intended to be a more powerful and safer alternative to reflect.DeepEqual for comparing whether two values are semantically equal. It is intended to only be used in tests, as performance is not a goal and it may panic if it cannot compare the values.